TestLib/EarlGreyImpl/GREYElementInteractionErrorHandler.m

```diff
@@ -78,7 +78,12 @@ void GREYHandleInteractionError(__strong GREYError *interactionError,
       id<GREYFailureHandler> failureHandler =
           [NSThread mainThread].threadDictionary[GREYFailureHandlerKey];
       // TODO(b/147072566): Will show up a (null) in rotation.
-      [failureHandler handleException:exception details:matcherDetails];
+      if ([reason containsString:@"the desired element was not found"]) {
```

Comment:
interactionError should be the raw error returned from the app side - is it not giving the ElementNotFoundErrorCode here?